<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
background-color: #131A20;</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="pScroll">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>798</width>
       <height>478</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1130</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <widget class="QPushButton" name="uart">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>210</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #0078D7;
text-align: center; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/uart.svg</normaloff>:/images/uart.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="can">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>350</y>
       <width>210</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #FFFFFF;
text-align: center; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/can.svg</normaloff>:/images/can.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="gpio">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>130</y>
       <width>210</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
color: black;
background-color: #FFFFFF;
text-align: bottom left; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string>GPIO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="music_player">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>240</y>
       <width>210</width>
       <height>210</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #EA3E3E;
text-align: center; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/music.svg</normaloff>:/images/music.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="sketchpad">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>210</width>
       <height>210</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #0078D7;
text-align: center; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/sketchpad.svg</normaloff>:/images/sketchpad.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="adc">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>240</y>
       <width>210</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #F2C811;
text-align: center; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/adc.svg</normaloff>:/images/adc.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="icm20608">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>210</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>90</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
color: black;
background-color: #FFFFFF;
text-align: bottom left; 
padding: 10px; }
</string>
     </property>
     <property name="text">
      <string>Icm20608</string>
     </property>
    </widget>
    <widget class="QPushButton" name="camera">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>210</width>
       <height>210</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #0078D7;
text-align: center;
padding: 10px; 
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/camera.svg</normaloff>:/images/camera.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ap3216c">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>210</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>90</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
text-align: center; 
background-color: #009E49;
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/ap3216c.svg</normaloff>:/images/ap3216c.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="ble">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>210</width>
       <height>210</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #FFFFFF;
text-align: center; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/ble.svg</normaloff>:/images/ble.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="config">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>20</y>
       <width>210</width>
       <height>210</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #DC3C00;
text-align: center; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/config.svg</normaloff>:/images/config.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="video">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>240</y>
       <width>210</width>
       <height>210</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
background-color: #0078D7;
text-align: center; 
padding: 10px; }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/video.svg</normaloff>:/images/video.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_uart">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>85</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: white; }</string>
     </property>
     <property name="text">
      <string>UART</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_adc">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>310</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: white; }</string>
     </property>
     <property name="text">
      <string>ADC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_config">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>190</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: white; }</string>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_adc_2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>425</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { color: black; }</string>
     </property>
     <property name="text">
      <string>CAN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recording">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>240</y>
       <width>210</width>
       <height>100</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>90</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { 
color: black;
background-color: #FFFFFF;
text-align: bottom left; 
padding: 10px; }
</string>
     </property>
     <property name="text">
      <string>Recording</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
